docs/content/reference/dynamic-configuration/kubernetes-crd-rbac.yml

```diff
@@ -0,0 +1,85 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: traefik-ingress-controller
+
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - secrets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - ingresses
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - ingresses/status
+    verbs:
+      - update
+  - apiGroups:
+      - traefik.containo.us
+    resources:
+      - middlewares
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
```

Comment:
Can this be combined to:
```yaml
  - apiGroups:
      - traefik.containo.us
    resources:
     - ingressroutes
     - ingressroutetcps
      - tlsoptions
      - middlewares
      - traefikservices
    verbs:
      - get
      - list
      - watch
```

and save like 50 lines?

Reply:
:laughing: